app/admin/product.rb

```diff
@@ -0,0 +1,17 @@
+ActiveAdmin.register Product do
+
+# See permitted parameters documentation:
+# https://github.com/activeadmin/activeadmin/blob/master/docs/2-resource-customization.md#setting-up-strong-parameters
+#
+# permit_params :list, :of, :attributes, :on, :model
```

Comment:
Let's get some real forms and things in here. Same for all the other models too. Ability to edit, delete and view and show information that are the most useful to the marketplace owner. Like for example when the marketplace owner is looking at a single shop page in the admin, show all the products in the shop, and in the products, show all the sales of those products, etc.